test/fake_3par_data.py

```diff
@@ -0,0 +1,376 @@
+CGSNAPSHOT_BASE_NAME = 'oss-6Rxe1druToSHJByeMeeh8g'
+CLIENT_ID = "12345"
+REPLICATION_CLIENT_ID = "54321"
+REPLICATION_BACKEND_ID = 'target'
+# fake host on the 3par
+FAKE_HOST = 'fakehost'
+FAKE_CINDER_HOST = 'fakehost@foo#' + HPE3PAR_CPG
+USER_ID = '2689d9a913974c008b1d859013f23607'
+PROJECT_ID = 'fac88235b9d64685a3530f73e490348f'
+VOLUME_ID_SNAP = '761fc5e5-5191-4ec7-aeba-33e36de44156'
+FAKE_DESC = 'test description name'
+FAKE_FC_PORTS = [{'portPos': {'node': 7, 'slot': 1, 'cardPort': 1},
+                  'type': 1,
+                  'portWWN': '0987654321234',
+                  'protocol': 1,
+                  'mode': 2,
+                  'linkState': 4},
+                 {'portPos': {'node': 6, 'slot': 1, 'cardPort': 1},
+                  'type': 1,
+                  'portWWN': '123456789000987',
+                  'protocol': 1,
+                  'mode': 2,
+                  'linkState': 4}]
+
+VVS_NAME = "myvvs"
+
+# Provisioning
+THIN = 'thin'
+FULL = 'full'
+DEDUP = 'dedup'
+
+FAKE_ISCSI_PORT = {'portPos': {'node': 8, 'slot': 1, 'cardPort': 1},
+                   'protocol': 2,
+                   'mode': 2,
+                   'IPAddr': '1.1.1.2',
+                   'iSCSIName': ('iqn.2000-05.com.3pardata:'
+                                 '21810002ac00383d'),
+                   'linkState': 4}
+volume = {'name': VOLUME_NAME,
+          'id': VOLUME_ID,
+          'display_name': 'Foo Volume',
+          'size': 2,
+          'host': FAKE_CINDER_HOST,
+          'provisioning': THIN,
+          'flash_cache': None,
+          'volume_type': None,
+          'volume_type_id': None,
+          'snapshots': []}
+
+snapshot1 = {'name': SNAPSHOT_NAME1,
+             'id': SNAPSHOT_ID1,
+             'parent_id': VOLUME_ID,
+             'expiration_hours': '10',
+             'retention_hours': '10'}
+
+snapshot2 = {'name': SNAPSHOT_NAME2,
+             'id': SNAPSHOT_ID2,
+             'parent_id': VOLUME_ID,
+             'expiration_hours': '5',
+             'retention_hours': '5'}
+
+snapshot3 = {'name': SNAPSHOT_NAME2,
+             'id': SNAPSHOT_ID2,
+             # This is a child of snapshot1
+             'parent_id': SNAPSHOT_ID1,
+             'expiration_hours': '5',
+             'retention_hours': '5'}
+
+volume_with_snapshots = {
+    'name': VOLUME_NAME,
+    'id': VOLUME_ID,
+    'display_name': 'Foo Volume',
+    'size': 2,
+    'host': FAKE_CINDER_HOST,
+    'provisioning': THIN,
+    'flash_cache': None,
+    'volume_type': None,
+    'volume_type_id': None,
+    'snapshots': [snapshot1, snapshot2]}
+
+
+volume_with_multilevel_snapshot = {
+    'name': VOLUME_NAME,
+    'id': VOLUME_ID,
+    'display_name': 'Foo Volume',
+    'size': 2,
+    'host': FAKE_CINDER_HOST,
+    'provisioning': THIN,
+    'flash_cache': None,
+    'volume_type': None,
+    'volume_type_id': None,
+    'snapshots': [snapshot1, snapshot2, snapshot3]}
+
+volume_replicated = {'name': VOLUME_NAME,
```

Comment:
can you remove this dictionary if its not needed

Reply:
Sure